tests/CMakeLists.txt

```diff
@@ -75,6 +75,8 @@ macro(add_unit_test)
         add_test(${_test_NAME} ${_test_LAUNCHER} ${_test_NAME})
     endif(KDE4_TEST_OUTPUT STREQUAL "xml")
 
+    set_tests_properties(${_test_NAME} PROPERTIES ENVIRONMENT "LANG=en_US")
```

Comment:
Maybe add `UTF-8` as suggested in https://github.com/keepassxreboot/keepassxc/issues/7747#issuecomment-1086740797 ?

Reply:
Makes sense. Updated to use `LANG=en_US.UTF-8`.